Core/MIPS/IR/IRInterpreter.cpp

```diff
@@ -82,7 +82,7 @@ u32 RunMemCheck(u32 pc, u32 addr) {
 template <uint32_t alignment>
 u32 RunValidateAddress(u32 pc, u32 addr, u32 isWrite) {
 	const auto toss = [&](MemoryExceptionType t) {
-		Core_MemoryException(addr, pc, t);
+		Core_MemoryException(addr, 4, pc, t);
```

Comment:
Actually, might be best to use `alignment`, since this is used for 16 byte reads as well.

-[Unknown]

Reply:
Ah, good point.